pipeline/src/pipeline/pipelinetypes.jl

```diff
@@ -22,3 +20,43 @@ The pipeline type for the Rt pipeline without renewal in test mode.
 """
 struct EpiAwareExamplePipeline <: AbstractEpiAwarePipeline
 end
+
+"""
+The pipeline type for the Rt pipeline for an outbreak scenario where Rt decreases
+    smoothly over time to Rt < 1.
+
+# Example
+
+```julia
+using EpiAwarePipeline, Plots
+pipeline = SmoothOutbreakPipeline()
+Rt = make_Rt(pipeline) |> Rt -> plot(Rt,
+    xlabel = "Time",
+    ylabel = "Rt",
+    lab = "",
+    title = "Smooth outbreak scenario")
+```
+"""
+struct SmoothOutbreakPipeline <: AbstractRtwithoutRenewalPipeline
```

Comment:
the fact we keep needing to make things abstract makes mee thing that everything should be abstract and then have a specific instance so extensions can be more easily made?